test/e2e/Deploy.t.sol

```diff
@@ -499,20 +476,13 @@ contract E2EDeploymentMainnetTest is DeployMainnet, CommonDeploymentTest {
       HAI_ETH_INITIAL_TICK // 2000 HAI = 1 ETH
     );
 
-    // Setup HAI/WETH oracle feed
-    IBaseOracle _haiWethOracle = uniV3RelayerFactory.deployUniV3Relayer({
-      _baseToken: address(systemCoin),
-      _quoteToken: address(collateral[WETH]),
-      _feeTier: HAI_POOL_FEE_TIER,
-      _quotePeriod: 1 days
-    });
-
-    // Setup HAI/USD oracle feed
-    systemCoinOracle = denominatedOracleFactory.deployDenominatedOracle({
-      _priceSource: _haiWethOracle,
-      _denominationPriceSource: delayedOracle[WETH].priceSource(),
-      _inverted: false
-    });
+    // NOTE: deploy [ UniV3 HAI/WETH + Chainlink ETH/USD ] oracle through governance actions
+    vm.startPrank(governor);
+    // grab the last denominated oracle deployed (in _setupPostEnvironment)
+    address[] memory _denominatedOracles = denominatedOracleFactory.denominatedOraclesList();
+    systemCoinOracle = IBaseOracle(_denominatedOracles[_denominatedOracles.length - 1]);
+    // assert we grabbed the correct oracle
+    assertEq(systemCoinOracle.symbol(), '(HAI / WETH) * (ETH / USD)');
```

Comment:
Regarding what the note says, I think that oracle has already been deployed during `_setupPostEnvironment`.